official/legacy/image_classification/configs/configs.py

```diff
@@ -92,12 +93,44 @@ class ResNetImagenetConfig(base_configs.ExperimentConfig):
   model: base_configs.ModelConfig = resnet_config.ResNetModelConfig()
 
 
+@dataclasses.dataclass
```

Comment:
Here are the pylints reported by CI:

FAIL: Found 24 non-whitelited pylint errors:
official/legacy/image_classification/vgg16/vgg_config.py:30: [E0001(syntax-error), ] invalid syntax

official/legacy/image_classification/configs/configs.py:42: [E0001(syntax-error), ] invalid syntax

official/legacy/image_classification/vgg16/vgg_model.py:33: [C0301(line-too-long), ] Line too long (81/80)

official/legacy/image_classification/vgg16/vgg_model.py:43: [C0301(line-too-long), ] Line too long (81/80)

official/legacy/image_classification/vgg16/vgg_model.py:56: [C0301(line-too-long), ] Line too long (81/80)

official/legacy/image_classification/vgg16/vgg_model.py:66: [C0301(line-too-long), ] Line too long (81/80)

official/legacy/image_classification/vgg16/vgg_model.py:79: [C0301(line-too-long), ] Line too long (81/80)

official/legacy/image_classification/vgg16/vgg_model.py:89: [C0301(line-too-long), ] Line too long (81/80)

official/legacy/image_classification/vgg16/vgg_model.py:99: [C0301(line-too-long), ] Line too long (81/80)

official/legacy/image_classification/vgg16/vgg_model.py:112: [C0301(line-too-long), ] Line too long (81/80)

official/legacy/image_classification/vgg16/vgg_model.py:122: [C0301(line-too-long), ] Line too long (81/80)

official/legacy/image_classification/vgg16/vgg_model.py:132: [C0301(line-too-long), ] Line too long (81/80)

official/legacy/image_classification/vgg16/vgg_model.py:145: [C0301(line-too-long), ] Line too long (81/80)

official/legacy/image_classification/vgg16/vgg_model.py:155: [C0301(line-too-long), ] Line too long (81/80)

official/legacy/image_classification/vgg16/vgg_model.py:165: [C0301(line-too-long), ] Line too long (81/80)

official/legacy/image_classification/vgg16/vgg_model.py:177: [C0301(line-too-long), ] Line too long (84/80)

official/legacy/image_classification/vgg16/vgg_model.py:184: [C0301(line-too-long), ] Line too long (84/80)

official/legacy/image_classification/vgg16/vgg_model.py:191: [C0301(line-too-long), ] Line too long (84/80)

official/legacy/image_classification/classifier_trainer_test.py:28: [C0301(line-too-long), ] Line too long (84/80)

official/legacy/image_classification/classifier_trainer_test.py:69: [C0326(bad-whitespace), ] No space allowed around keyword argument assignment

official/legacy/image_classification/classifier_trainer_test.py:102: [C0326(bad-whitespace), ] No space allowed around keyword argument assignment

official/legacy/image_classification/classifier_trainer_test.py:103: [C0326(bad-whitespace), ] No space allowed around keyword argument assignment

official/legacy/image_classification/classifier_trainer.py:134: [C0326(bad-whitespace), ] No space allowed around keyword argument assignment

official/legacy/image_classification/classifier_trainer.py:425: [C0326(bad-whitespace), ] No space allowed around keyword argument assignment

Could you fix them? 
Note that, pylint can be wrong so if you feel any does not make sense, please ignore them

Reply:
Yes, just fixed them. The following I couldn't find a solution, so I just ignored them like you said:

official/legacy/image_classification/classifier_trainer_test.py:102: [C0326(bad-whitespace), ] No space allowed around keyword argument assignment

official/legacy/image_classification/classifier_trainer_test.py:103: [C0326(bad-whitespace), ] No space allowed around keyword argument assignment

official/legacy/image_classification/classifier_trainer.py:134: [C0326(bad-whitespace), ] No space allowed around keyword argument assignment

official/legacy/image_classification/classifier_trainer.py:425: [C0326(bad-whitespace), ] No space allowed around keyword argument assignment